packages/server/src/profile/profile.controller.ts

```diff
@@ -81,4 +95,39 @@ export class ProfileController {
 
     return this.get(user);
   }
+
+  @Post('/upload_picture')
+  @UseInterceptors(
+    FileInterceptor('file', {
+      dest: 'uploads/',
+    }),
+  )
+  async uploadImage(
+    @UploadedFile() file,
+    @User() user: UserModel,
+  ): Promise<void> {
+    if (user.photoURL) {
+      fs.unlinkSync('uploads/' + user.photoURL);
```

Comment:
`sync` filesystem functions are slow af. pls await the async versions so our poor CPU can do other stuff while the hard drive seeks